pkg/psmdb/backup/pbm.go

```diff
@@ -48,6 +49,10 @@ const (
 	KMSKeyID                         = "KMS_KEY_ID"
 )
 
+const (
+	BackupTypeIncrementalBase defs.BackupType = defs.IncrementalBackup + "-base"
+)
+
```

Comment:
We can also move this type to `pkg/apis/psmdb/v1/perconaservermongodbbackup_types.go` since it is more related to 

```
type PerconaServerMongoDBBackupSpec struct {
	PSMDBCluster     string                   `json:"psmdbCluster,omitempty"` // TODO: Remove after v1.15
	ClusterName      string                   `json:"clusterName,omitempty"`
	StorageName      string                   `json:"storageName,omitempty"`
	Compression      compress.CompressionType `json:"compressionType,omitempty"`
	CompressionLevel *int                     `json:"compressionLevel,omitempty"`

	// +kubebuilder:validation:Enum={logical,physical,incremental}
	Type defs.BackupType `json:"type,omitempty"`
}

```